website/content/en/docs/contribution-guidelines/developer-guide.md

```diff
@@ -1,97 +1,62 @@
-
-```sh
-$ make test
-```
-
-For more information on running testing and correctly configuring your environment,
-refer to the [`Running the Tests Locally`][running-the-tests] document. To incorporate code changes in your development environment see the [`Testing changes Locally`][testing-changes-locally] document.
-
-To run the lint checks done in the CI locally, run:
-
-```sh
-$ make lint
-```
-
-**NOTE** Note that for it is required to install `golangci-lint` locally. For more info see its [doc](https://github.com/golangci/golangci-lint#install)
-
-## How the operator-sdk binaries are built
-
-In the release process, the script [.release.sh][release-sh] is executed and it will call the [makefile][makefile] target [make release](https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L113). To know more about the release process, see the [doc][release-doc] also see [operator-sdk releases](https://github.com/operator-framework/operator-sdk/releases).
-
-**NOTE** The Deploy stage (configured in [.travis.yml ][travis]) builds also execute the same [makefile][makefile] targets. This stage is executed against the master branch when a Pull Request is merged.
-
-## How to test the build of operator-sdk binaries
-
-Follow these steps to execute the Travis `Deploy` stage against your branch to demonstrate that the merge build will complete as expected.
+See the [testing][dev-testing] and [documentation][dev-docs] guides for more information.
 
-- Enable the Travis in your fork repository. For more info see [`To get started with Travis CI using GitHub`](https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci-using-github) 
-- Create image repos in quay (or another registry that supports multi-arch images) for ansible, helm, and scorecard proxy. For each image type, you need repos for the manifest list and one for each architecture (e.g. `ansible-operator`, `ansible-operator-amd64`, `ansible-operator-s390x`, etc.)
+## Releasing
 
-**NOTE** Be sure to make each repository public.
+See the [release guide][dev-release] for more information.
 
-- Set the following environment variables in the Travis settings for your fork:
+## Continuous Integration (CI)
 
-    - `ANSIBLE_IMAGE` docker image name (e.g. `quay.io/joelanford/ansible-operator`)
-    - `HELM_IMAGE` same as above, but for helm
-    - `SCORECARD_PROXY_IMAGE` same as above, but for scorecard proxy
-    - `DOCKER_USERNAME` credentials for your repo
-    - `DOCKER_PASSWORD` credentials for your repo
-    - `DOCKER_CLI_EXPERIMENTAL`  set to `enabled` 
-    - `COVERALLS_TOKEN`  token to integrate the project with `https://coveralls.io/`. So, enable your fork in `https://coveralls.io/` and generate a token to allow it. 
+The operator-sdk repo uses [Travis CI][travis] to test each pull request and build images for both master commits
+and releases. You can alter these processes by modifying this [`.travis.yml`][travis.yml] config file.
```

Comment:
isn't this kind of weird advice? If i'm going to be maintaining a fork, i don't want to have a floating commit that I need to keep on top with my .travis.yaml changes, no?

Reply:
Why would you need a special commit for your fork? You are making these changes with the intent of them being in SDK master, so once that is done you ff your fork.

Reply:
Oh, I misunderstood what you meant by "You can alter these processes..."